readthedocs/search/parsers.py

```diff
@@ -67,6 +67,7 @@ def _get_page_content(self, page):
                 type_="html",
                 version_slug=self.version.slug,
                 include_file=False,
+                version_type=self.version.type,
```

Comment:
This is because previously we weren't running search on PR builds, right? And now the FTD is only on PR builds? 😆

Reply:
yeah, we weren't reading the content of the files from PR previews, and now we do.